0000-report-inboxes.md

```diff
@@ -0,0 +1,82 @@
+- Feature Name: report-inboxes
+- Start Date: 2024-02-20
+- RFC PR: [LemmyNet/rfcs#0000](https://github.com/LemmyNet/rfcs/pull/0000)
+- Lemmy Issue: [LemmyNet/lemmy#0000](https://github.com/LemmyNet/lemmy/issues/0000)
+
+# Summary
+
+Rather than combining all reports into a single report inbox, we should allow users to select whether they are reporting to mods or admins, and we should split reports into different inboxes based on that selection.
+
+# Motivation
+
+The current approach has some shortcomings:
+
+* Users are not currently able to bypass mods and report directly to admins - this may allow mods to conceal instance rule breaking in specific communities
+* Admins are not aware of community rules, so they may wish to take no action for most community rule breaking reports. However, if an admin resolves such a report, the relevant community mods most likely never see it.
+* Different instances may have different rules, but somebody resolving a report on one instance will resolve it for other instances as well, thus potentially resulting in missed reports.
+* Mods might take local action on a report and mark it as resolved even in cases where a user should be banned from the entire instance. In this case, admins are very unlikely to see the report.
+
+# Guide-level explanation
+
+### When creating reports, users will be able to select if it's a mod report, or an admin report (or both)
+
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/9a21b527-6c88-4024-b287-3371d77688f4)
+
+**Note: labels on the sreenshot are illustrative, actual labels can be more user-friendy.** Maybe something like:
+* Breaks community rules (report sent to moderators)
+* Breaks instance rules (report sent to admins)
+
+### Instead of the current single report inbox, there will be three different kinds of inboxes
+
+* Admin reports - show all reports sent to admins (only visible to admins) 
+* Mod reports - show all reports sent to mods for any communities the user moderates (visible to admins in case they are explicit mods in any communities)
+   * This is equivalent to the report view that mods currently have in Lemmy already
+* All reports - Shows a view of all (admin and mod) reports, only visible to admins
+   * This is akin to the current 0.19.3 admin report view, and would allow admins to still keep an eye on mod actions on their instance if they wish
+    
+The UI wouldn't need to change for mods, but for admins, there would be a new selection at the top of the reports page (the "mod reports" tab would only be visible if the admin is also a mod in any community):
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/cc4ad68c-6e85-4cd9-b324-131c06951cb3)
+
+### Resolving reports should be more granular
```

Comment:
I had to read this over a few times to fully understand it. Might be good to break this into 3 sub-headings. And what resolving, or removing does to the other inboxes.

`#### Reports to Admins`

`#### Reports to Mods`

`#### Reports to Both`